### DISCUSSION TITLE
Linux服务器无法正常访问端口，已关闭防火墙等

### DISCUSSION THREAD

**Discussion by gfr5728** [AlistGo/alist#2509] [Q&A]
自建Linux服务器自动安装后启动服务却无法从5244端口访问，卸载后手动安装同样也不起效果。防火墙和本地安全策略组已关闭，其他端口如默认80端口可正常访问，请问需要怎么解决？重装多次也无济于事..

![~5PZ{V2M1$ZG5B4%@FLOVWG](https://user-images.githubusercontent.com/67219284/204139244-49f248ee-d9ec-4e80-8fec-7230cf634dec.png)
![FHW)KB1AFGH%T(2O GRHZCY](https://user-images.githubusercontent.com/67219284/204139250-ffd7f662-aede-4294-b881-d9ba7b4169bd.png)

**Comment by PanDaDa0202** [comment]
[我今天也遇到了这个问题，卸载重装alist解决了，但是又遇到无法挂载的问题
[http://cdn.img.kggzs.cn/uploads/img/2022/47/202263836ec76d2d6.png](url)

**Reply by PanDaDa0202** [reply]
`[img]http://cdn.img.kggzs.cn/uploads/img/2022/47/202263836ec76d2d6.png[/img]`

**Reply by gfr5728** [reply]
卸载重装了好几遍也不行啊……
